### ISSUE TITLE
UTF8 file is getting parsed

### ISSUE THREAD

**Issue by arshadabm** [parsecsv/parsecsv-for-php#200]
I have a file with Turkish symbols e.g İlçe, Döşemealtı, Şereflikoçhisar but when I parse this file, it return empty.
I am using this code
```$csv = new \ParseCsv\Csv();
$csv->encoding('UTF-16', 'UTF-8');
$csv->delimiter = ",";
$csv->parseFile('cities.txt');
print_r($csv->data);```

**Comment by Fonata** [comment]
Hi @arshadabm, thanks for opening an issue here! :wave:  

Please provide a link to an example file. To get a link you can store your file in a cloud storage like Dropbox or OneDrive. Make sure that it requires no login and doesn't contain any sensitive information.

**Comment by arshadabm** [comment]
Hi @Fonata 
I have uploaded the file in dropbox. 
https://www.dropbox.com/s/q18lz819mgrre0u/cities.txt?dl=0

This file is encoded to utf-8 CSV

**Comment by Fonata** [comment]
I found two issues with your code example above:
- The of the encodings is expected to be input, output. I think your line should be `$csv->encoding('UTF-8', 'UTF-16');`
- The currently released version of ParseCsv doesn’t set `$csv->data` when used in the way above. I will create a pull request for the community to review that fixes this. You should see the parsed data when you change your call to `print_r($csv->parseFile('cities.txt'));`

**Comment by Fonata** [comment]
I have now released [version 1.3.1](https://github.com/parsecsv/parsecsv-for-php/releases/tag/1.3.1) of this library, fixing this issue.
